scripts/zones/Mamook/mobs/Gulool_Ja_Ja.lua

```diff
@@ -4,31 +4,63 @@
 -----------------------------------
 
 require("scripts/globals/titles");
-
------------------------------------
--- onMobSpawn Action
------------------------------------
+require("scripts/globals/status");
+require("scripts/zones/Mamook/MobIDs");
 
 function onMobSpawn(mob)
+    mob:setLocalVar("1Hour", math.random(3,5)); 
+    mob:setMod(MOD_DOUBLE_ATTACK, 20);
 end;
 
------------------------------------
--- onMobEngaged Action
------------------------------------
-
 function onMobEngaged(mob,target)
+    for i = 17043876, 17043879, 1 do
+        SpawnMob(i):updateEnmity(target);
+    end
+end;
 
-    SpawnMob(17043876):updateEnmity(target);
-    SpawnMob(17043877):updateEnmity(target);
-    SpawnMob(17043878):updateEnmity(target);
-    SpawnMob(17043879):updateEnmity(target);
+function onMobFight(mob, target)
+    local petIDs = {17043876,17043877,17043878,17043879};
+    local petStatus = {GetMobAction(petIDs[1]),GetMobAction(petIDs[2]),GetMobAction(petIDs[3]),GetMobAction(petIDs[4])};
```

Comment:
we're phasing out global function GetMobAction.  Instead, use ... {cont'd below}